src/main/java/com/rakovets/course/java/core/practice/enum_types/Seasons.java

```diff
@@ -0,0 +1,8 @@
+package com.rakovets.course.java.core.practice.enum_types;
+
+public enum Seasons {
```

Comment:
"Violation: Contains code not related to this task
Solution: Remove code not related to this task"